parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -264,6 +270,50 @@ try
     . c:\AzureData\windows\kubernetesfunc.ps1
     . c:\AzureData\windows\nvidiagpudriverfunc.ps1
 
+    $KubeletConfigArgsStr=$global:KubeletConfigArgs -join " "

```

Comment:
Please do not add the logic in this file directly since the customdata has the length limitation.
It seems like that you can add the logic as a function in staging/cse/windows/kubernetesfunc.ps1

Reply:
This PR is to add them to cse package https://github.com/Azure/AgentBaker/pull/4281/files